kolibri/core/assets/src/views/SidePanelModal/index.vue

```diff
@@ -66,15 +66,14 @@
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
```

Comment:
Looks like there's already a copy of the SidePanelModal in kolibri-common, but it may have been altered. This may require some reconciliation to have it work as expected here and elsewhere.

Reply:
With a small tweak iin one of the learn elements this SidePanelModal can be used here too without more changes. Done!